combat-trainer.lic

```diff
@@ -416,6 +416,9 @@ class LootProcess
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
 
+    @gem_pouch_adjective = settings.gem_pouch_adjective
```

Comment:
❓ Is this a duplicate of line 406?

It looks like this adds the setting to `LootProcess` (for a second time), but the new use of the setting is in `TrainerProcess`. I would have expected to see these two lines added around line 1989 instead.